tool/ci.dart

```diff
@@ -268,8 +268,9 @@ class ExampleTask extends Task {
     required List<String> packages,
     required ArgResults argResults,
   }) async {
-    const examplesWithTest = [
+    final examplesWithTest = [
```

Comment:
Why this change?

Reply:
Oh, I had a platform guard there while developing. Nice catch!